### DISCUSSION TITLE
Lesson 9. Why my anvil don't stop?

### DISCUSSION THREAD

**Discussion by LareineHan** [Cyfrin/foundry-full-course-cu#1343] [Q&A] [answered]
Hi all,
When I run `anvil` or `make anvil` either way, I see that block is hashing continuously. It just keeps continuing, and I can't stop or kill it. What is the problem, and how can I fix this? 
I am seeing this pattern over and over. 
<img width="669" alt="Screenshot 2024-01-19 at 4 25 45 PM" src="https://github.com/Cyfrin/foundry-full-course-f23/assets/128317012/95e3fbf1-6fd3-4a3d-b53d-94c9f6da9eaf">

**Comment by xyz899** [comment]
Hi, 

This is an uncommon error to have, have you tried the following basics troubleshooting ? :

1. `foundryup`
2. Restart VSCode
3. Restart your Machine
4. Checked if your terminal is up to date
5. Check if the network is stable

Because technically the block you see every time is supposed to be an option called blocktime 
```
-b, --block-time <SECONDS>
          Block time in seconds for interval mining
          [aliases: blockTime]
```
in --help try to set it to 0 or a very big integer and see what it does

**Comment by shikhar229169** [comment, marked as answer]
This is actually not a problem.
This means that your anvil is configured to mine a block every second.
I can see that in the Makefile, anvil is configured to mine every block per second:
https://github.com/Cyfrin/foundry-defi-stablecoin-f23/blob/main/Makefile#L36
Also, you can refer to the foundry docs: https://book.getfoundry.sh/reference/anvil/?#mining-modes
